partner_financial_risk/models/res_partner.py

```diff
@@ -56,6 +56,7 @@ class ResPartner(models.Model):
         help='It Indicate if partner risk exceeded')
     credit_policy = fields.Char()
     risk_allow_edit = fields.Boolean(compute='_compute_risk_allow_edit')
+    credit_limit = fields.Float(track_visibility='onchange')
```

Comment:
🤔 Do you add a field and nothing more? Don't you add it to a view or something like that?

Reply:
This field already exists before.